src/main/telemetry/frsky.h

```diff
@@ -17,6 +17,47 @@
 
 #pragma once
 
+#include "common/time.h"
+
+#if defined(TELEMETRY_FRSKY)
+#if defined(GPS)
+    #define TELEMETRY_FRSKY_CONFIG_S \
```

Comment:
The contents of parameter groups may not be conditional. One intention behind the introduction of parameter groups is to allow for them to be used to transmit configuration settings between the firmware and the configurator (or any other configuration tool). This requires any given parameter group with a given version to be identical for all targets (apart from potentially missing values at the end if they have been added later on).

Reply:
Ok i will revert PG changes